### DISCUSSION TITLE
Unable to access argo workflows after installation (helm) with admin rights

### DISCUSSION THREAD

**Discussion by S0PEX** [argoproj/argo-workflows#9142] [Q&A] [answered]
Hey,

I have just installed argo workflow through helm and having issues with permissions.
Basically I want to have a user with all permission for testing so that I can access all rest endpoints.

What I did is basically:
```shell
$ helm upgrade --install argo-workflow argo-workflows \
  --set server.secure=true \
  --repo https://argoproj.github.io/argo-helm \
  --namespace argo-workflow --create-namespace
```

Create Admin role
```shell
$ kubectl create rolebinding default-admin --clusterrole=admin --serviceaccount=argo-workflow:default -n argo-workflow
```

Obtain JWT:
```shell
$ SECRET=$(kubectl get sa default -o=jsonpath='{.secrets[0].name}'  -n argo-workflow) 
ARGO_TOKEN="Bearer $(kubectl get secret $SECRET -o=jsonpath='{.data.token}' -n argo-workflow | base64 --decode)"
$ echo $ARGO_TOKEN
```

I then login into the panel with the received Bearer token but get the error that my user doesn't have the right permission.
e.g ```Forbidden: workflows.argoproj.io is forbidden: User "system:serviceaccount:argo-workflow:default" cannot list resource "workflows" in API group "argoproj.io" at the cluster scope```

Could somebody hint me what I am doing wrong?

Thanks.

Regards,
Artur

**Comment by terrytangyuan** [comment, marked as answer]
Your SA may not have permission at the cluster scope.

**Reply by agustinvinao** [reply]
It will help if you can share a link or anything to help to solve it, the problem is clear.
